eventstream.go

```diff
@@ -186,10 +215,10 @@ func (e *EventStream) SubscriberAggregateType(f func(e Event), aggregates ...Agg
 	return &s
 }
 
-// SubscriberSpecificEvent bind the f function to be called on specific events
-func (e *EventStream) SubscriberSpecificEvent(f func(e Event), events ...interface{}) *Subscription {
-	s := Subscription{
-		f: f,
+// SpecificEvent subscribe on specific events where the interface is a pointer to the event struct
+func (e *EventStream) SpecificEvent(f func(e Event), events ...interface{}) *subscription {
```

Comment:
Is there a better name for this method, because I feel like `SpecificEvent` sounds to generic. What I mean is that there's an ambiguity between other subscribe methods because, for instance, `EventStream.Name` can also be considered a specific event (i.e. an event with a specific _name_).

Reply:
The difference between the `SpecificEvent` and the `Name` is that the former takes a typed event as input, where in the Name the input is strings making it possible to subscribe to events without the reference to the specific types.

Reply:
In the `Name` you also have to supply the aggregate name `Name(f func(e Event), aggregate string, events ...string)`. To make sure event with the same name but from different contexts are not being subscribed to.

Reply:
Alright, but could `SpecificEvent` be called `Typed` instead?

Reply:
I have change it to only `Event`

Reply:
I have changed the names on almost all subscriber methods to try make them more consistent and easier to understand. I probably have to add more docs. But can you understand what each subscriber is doing?